src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveCourtVenuesBySearchStringIntegrationTest.java

```diff
@@ -60,7 +76,8 @@ void shouldRetrieveCourtVenues_For_SearchString_And_CourtTypeId_WithStatusCode_2
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"?search-string=zz&court-type-id=1000", "?search-string=AB$&court-type-id=1000",
+    @ValueSource(strings = {"?search-string=abc--", "?search-string=ab__c", "?search-string=___c",
+        "?search-string=zz&court-type-id=1000", "?search-string=AB$&court-type-id=1000",
         "?search-string=AB$&court-type-id=1,2,$,4"})
```

Comment:
can you add other search strings to this test case with only repeated special chars like 3 underscores and likewise other all allowed special chars, please ?

Reply:
sure, I'll add them.

Reply:
addressed [7a3ac72](https://github.com/hmcts/rd-location-ref-api/pull/473/commits/7a3ac72a8c9d5afffdb33a70158fc51a14a52465)